src/melee.cpp

```diff
@@ -2273,6 +2251,39 @@ std::vector<special_attack> player::mutation_attacks(Creature &t) const
 
 std::string melee_message( const ma_technique &tec, player &p, const dealt_damage_instance &ddi )
 {
+    // Those could be extracted to a json
+
+    // Three last values are for low damage
+    static const std::array<std::string, 6> player_stab = {{
+        "You impale %s", "You gouge %s", "You run %s through",
```

Comment:
The strings are not extracted for translation, so attempting to translate them at the end of this function will properly do nothing.